### PR TITLE
Scholarlify-BPP-Financial Support- API Payloads

### PR DESCRIPTION
**Pull request by nikhil-scholarlify**

### CODE REVIEW

File: logs/Scholarlify-BPP-Financial Support/on_search-response.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "bap_id": "fs-ps-bap-network.onest.network",
+    "bap_uri": "https://fs-ps-bap-network.onest.network",
+    "transaction_id": "41bcfd01-da1c-4c4d-b39e-4418779aea80",
+    "message_id": "ac1756d1-0f44-4321-8abc-3509941ad921",
+    "ttl": "PT10M",
+    "timestamp": "2024-05-28T08:41:54.199Z"
+  },
+  "message": {
+    "ack": {
+      "status": "ACK"
```
Comment (shiva-rakshith) [inline logs/Scholarlify-BPP-Financial Support/on_search-response.json]: Ack response is not required. These JSON files can be removed from this folder

File: logs/Scholarlify-BPP-Financial Support/on_init-step1.json [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+                "name": "Application Form"
+              },
+              "index": {
+                "min": 0,
+                "cur": 0,
+                "max": 0
+              },
+              "headings": [
+                "Application Details",
+                "Review & Submit"
```
Comment (shiva-rakshith) [inline logs/Scholarlify-BPP-Financial Support/on_init-step1.json]: As there is only one form. Review and submit can be removed

File: logs/Scholarlify-BPP-Financial Support/on_search-response.json (hunk 1 above)
Comment (nikhil-scholarlify) [reply logs/Scholarlify-BPP-Financial Support/on_search-response.json]: ok removed the response files

File: logs/Scholarlify-BPP-Financial Support/on_init-step1.json (hunk 2 above)
Comment (nikhil-scholarlify) [reply logs/Scholarlify-BPP-Financial Support/on_init-step1.json]: ok updated to remove "Review & Submit"

**Review by shiva-rakshith — APPROVED** [review]

**Pull request merged**